testenrichers/cdi/pom.xml

```diff
@@ -46,9 +46,9 @@
     </dependency>
 
     <dependency>
-      <groupId>javax.enterprise</groupId>
-      <artifactId>cdi-api</artifactId>
-      <version>1.0</version>
+      <groupId>jakarta.enterprise</groupId>
+      <artifactId>jakarta.enterprise.cdi-api</artifactId>
+      <version>2.0.2</version>
```

Comment:
Can this be defined as `${version.jakarta-ee}` or something similar?

Reply:
version.jakarta.cdi-api added